packages/react-components/src/components/utils/GridLayoutUtils.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { BlockProps } from '../GridLayout';
+
+const TARGET_RATIO = 16 / 9;
+const TOLERANCE_RATIO = 8 / 9;
+
+const isRatioBetterThan = (A: number, B: number, targetRatio?: number): boolean => {
+  const _targetRatio = targetRatio ?? TARGET_RATIO;
+  return Math.abs(_targetRatio - A) < Math.abs(_targetRatio - B);
+};
+
+/**
+ * @private
+ */
+export const calculateBlockProps = (n: number, width: number, height: number): BlockProps => {
+  if (width <= 0) {
+    throw Error('Width provided [' + width + '] is less than or equal to 0.');
+  } else if (height <= 0) {
+    throw Error('Height provided [' + height + '] is less than or equal to 0.');
+  }
+  const areaRatio = width / height;

```

Comment:
`aspectRatio` is the standard term used for this. Any particular reason why this is called `areaRatio`?